test/corpus/insert.txt

```diff
@@ -526,3 +526,38 @@ FROM
                     (identifier))
                   (literal)))))
           (identifier))))))
+
+================================================================================
+Insert from unioned select
+================================================================================
+
+INSERT INTO some_table
+  (field)
+  (SELECT "String value"
+   UNION
+   SELECT "String value");
```

Comment:
For pg these should be single quotes, right?

Reply:
Yes

Reply:
yeah, double quotes are a MySQL/MariaDBism afaik